eclipse-theia/theia
# Replace `nsfw` with `@parcel/watcher`

#### What it does

Continuation of https://github.com/eclipse-theia/theia/pull/12784

Closes https://github.com/eclipse-theia/theia/issues/14015

Replaces our `nsfw` dependency with `@parcel/watcher` to align with vscode.

#### How to test

Test file watching related capabilities and confirm that there are no regressions:
* Editing files on the file system should be reflected in the editor
* Editing `settings.json` should be reflected in changes in the preferences
* Editing `launch.json` should be reflected in the debug view

#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
